root/main/ConfirmSeed.js

```diff
@@ -50,7 +61,7 @@ const ConfirmSeed = ({
       </p>
       <form method="post">
```

Comment:
Would it be better to remove query parameters here and set the `action` attribute to the bare path?

Reply:
It shouldn't cause any issues to submit with `skip_confirmation` included AFAIK.